README.md

```diff
@@ -20,6 +20,8 @@ This is the codebase for the /tg/station flavoured fork of SpaceStation 13.
 
 Space Station 13 is a paranoia-laden round-based roleplaying game set against the backdrop of a nonsensical, metal death trap masquerading as a space station, with charming spritework designed to represent the sci-fi setting and its dangerous undertones. Have fun, and survive!
 
+*All github inquiries (such as moderation actions) may be handled via the /tg/station discord #coding-general. Simply ping @maintainers, following the channel description asking questions guide, with your issue!*
```

Comment:
```suggestion
*All github inquiries (such as moderation actions) may be handled via the /tg/station discord #coding-general. Simply ping the `@Maintainer` role, following the guide in the channel's description for asking questions, with your issue!*
```

fixed grammar. need to have `@Maintainer` wrapped otherwise it'll redirect to whoever owns that github handle